addons/survey/models/survey_question.py

```diff
@@ -553,6 +553,9 @@ class SurveyQuestionAnswer(models.Model):
     value_image = fields.Image('Image', max_width=256, max_height=256)
     is_correct = fields.Boolean('Is a correct answer')
     answer_score = fields.Float('Score for this choice', help="A positive score indicates a correct choice; a negative or null score indicates a wrong answer")
+    question_type = fields.Selection(related='question_id.question_type')
```

Comment:
I would move those under the "question_id" field and create a small section with a python comment.

Something like:

```py
# question and question related fields
question_id = fields.Many2one('survey.question', string='Question', ondelete='cascade')
question_type = fields.Selection(related='question_id.question_type')
scoring_type = fields.Selection(related='question_id.scoring_type')
allow_value_image = fields.Boolean(related='question_id.allow_value_image')
# answer specific fields
...
```

It's usually good to give some structure when we start having a lot of fields.